maxim/tests/test_fireworks.py

```
@@ -472,7 +472,7 @@
         if not fireworksApiKey:
             raise ValueError("FIREWORKS_API_KEY environment variable is not set")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)